output/telegram.go

```diff
@@ -75,6 +84,7 @@ func (t *TelegramOutput) sendMessage(spanCtx sreCommon.TracerSpanContext, IDToke
 
 	if err != nil {
 		t.logger.SpanError(span, err)
+		t.logger.SpanError(span, message)
```

Comment:
May be better log once err+message?

Reply:
typo. should read 
t.logger.SpanDebug(span, message)